module-4/backend/week_two.md

```diff
@@ -4,9 +4,35 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-1. What is Webpack and why is it useful?
-2. When do you want to use event delegation?
+1. What is Webpack and why is it useful? 
+  - A frontend frame work to allow a platform for a server, and webtessting. Its useful because you can then update the application with live changes 
+2. When do you want to use event delegation? 
+  - Any time you "go" out to do something that takes more time. Like an api reqpuest
 3. What's one difference between ES5 and ES6?
+  - interpolation - ${}
+4. What's the deal with semi-colons in JavaScript? 
+  - You don't need them with es6 unless you start a line with an opening bracket
+5. How are you using the MVC design pattern in your Quantified Self project? 
+  - the route file directs corresponding routes to the method of the controller. There it then gathers all necessary information including model information. In the model file is where database information is collected
+6. How do you execute raw SQL in node? 
+  - useing knex.raw 
+7. What's a callback function and what are some reasons when we use/need callback functions? 
+- a callback function is something we use when geting information from an api and its handed to the funciton that does something with it
+8. What is CORS?
+ - cross origin resource sharing - using two programs fors the same project
+9. What are some steps to avoid CORS?
+ - using the same project for databases and frontend display. 
```

Comment:
you can do that, but you can't always control this. you may need to use a gem or a package to help with this